### PR TITLE
Feature/symfony3 compatibility

### PR DESCRIPTION
**Pull request by lenybernard**
## Type
Symfony3 compatibility

## Purpose
This PR brings support of Symfony v3, Symfony 2.8 is still supported

## BC Break
YES... NO actually I don't think so ?

### CODE REVIEW

**Comment by paulandrieux** [comment]
You have some behat errors I fixed (there)[https://github.com/paulandrieux/victoire/commits/feature/sf3.3]

My Branch is from the upstream master branch, so I can't rebase it on yours. 
However, I started a (PR to update master with 2.3)[https://github.com/Victoire/victoire/pull/926], so when this PR will be merged, my fixes will be easier to put onto your PR

File: composer.json [hunk 1]
```diff
@@ -44,13 +45,15 @@
...
+        "sensio/framework-extra-bundle": "^3.0.2",
         "snc/redis-bundle": "~2.0",
         "stof/doctrine-extensions-bundle": "~1.2",
         "symfony/assetic-bundle": "~2.3",
         "symfony/monolog-bundle": "~2.4|~3.0",
-        "symfony/symfony": "^2.8",
-        "symfony/swiftmailer-bundle": "^2.3",
+        "symfony/polyfill-apcu": "^1.0",
+        "symfony/swiftmailer-bundle": "^2.3.10",
+        "symfony/symfony": "~3.0",
```
Comment (alexislefebvre) [inline composer.json]: We have to keep Symfony 2.8 compatibility here: `"symfony/symfony": "~2.8|~3.0",`

**Comment by alexislefebvre** [comment]
Do we still run tests with Symfony 2.8? I had a old PR about restoring Travis CI builds: https://github.com/alexislefebvre/victoire/pull/14 Even if it was slow, it allowed to test several versions of Symfony. I'll try to update it with the code from this PR.

**Comment by lenybernard** [comment]
Guys @alexislefebvre @paulandrieux @gregumo, I need a final review before merging with the swag we deserve :p

File: composer.json [hunk 2]
```diff
@@ -87,7 +90,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.3.x-dev"
+            "dev-feature/symfony3-compatibility": "2.3.x-dev"
```
Comment (paulandrieux) [inline composer.json]: I think we do not need any branch alias for 2.x branches. A branch alias is required only on the master branch to match the next major release (3.0 ATM).

**Pull request merged**